src/main/kotlin/day11/ExceptionExample.kt

```
@@ -0,0 +1,18 @@
+package org.example.day11
+
+fun main() {
+    val numString = "10.5"
+    val num: Int = try {
+        numString.toInt()
+    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)